core-web/libs/dotcms-scss/angular/_mixins.scss

```diff
@@ -76,7 +76,7 @@
 
 @mixin dot-field-helper-on-input {
     position: absolute;
-    top: $spacing-5;
+    top: 1.8rem;
```

Comment:
is this intended?

Reply:
@zJaaal Should we create a variable for that `arbitrary` value?

Reply:
We surely can. I'll create it